### PR TITLE
Add workflow for product V2 sanity tests

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Add workflow for product V2 sanity tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Related PRs       | 
| How to test?      | New workflow for product V2 tests should run and be green
| Possible impacts? | none

### CODE REVIEW

File: .github/workflows/sanity-productV2.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+on:
+  pull_request:
+    paths:
+      - .github/workflows/sanity-productV2.yml
+      - admin-dev/themes/new-theme/js/pages/product/**
+      - src/PrestaShopBundle/Form/Admin/Sell/Product/**
+      - src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/**
+      - src/Core/Domain/Product/**
+      - src/Adapter/Product/**
+      - src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/**
```
Comment (jolelievre) [inline .github/workflows/sanity-productV2.yml:12]:
```suggestion
      - src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/**
      - tests/UI/campaigns/productV2/**
```

Shouldn't it watch the test campaigns as well?

Comment (boubkerbribri) [reply .github/workflows/sanity-productV2.yml:12]: we could yes

**Review by jolelievre — DISMISSED** [review]
Thanks @boubkerbribri

**Review by nesrineabdmouleh — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by yanmakouf — DISMISSED** [review]
🙏

**Comment by boubkerbribri** [comment]
Install Shop option is working now so should be good on CI

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Sanity Tests are always :red_circle:

**Comment by boubkerbribri** [comment]
> Sanity Tests are always red_circle

Tests fixed on another PR, tests should be green now. Waiting fo CI to confirm

**Review by Progi1984 — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Pull request merged**